tox.ini

```diff
@@ -43,7 +43,7 @@ deps = -r{toxinidir}/requirements.txt
 
 [flake8]
 #ignore = E122,E125,E126,E128,E129,E251,E265,E713,F402,F811,F812,H104,H237,H302,H304,H305,H307,H401,H402,H404,H405,H904
-ignore = W504,W503,H202,H401,H404,H405
+ignore = W504,W503,H202,H401,H404,H405,H216
```

Comment:
Why was this added?

Reply:
For each import mock, in travis logs getting below msg,
```
H216: The unittest.mock module should be used rather than the third party mock package unless actually needed. If so, disable the H216 check in hacking config and ensure mock is declared in the project's requirements.
225 import mock
```

Done, Discussed over teams